tmt/schemas/common.yaml

```diff
@@ -334,3 +334,26 @@ definitions:
   # https://tmt.readthedocs.io/en/stable/spec/plans.html#multihost
   where:
     $ref: "/schemas/common#/definitions/one_or_more_strings"
+
+  result_outcome:
```

Comment:
hmm, should this not be somewhere documented? What result outcomes does a test have?

Reply:
I believe this is covered under the [execute](https://tmt.readthedocs.io/en/stable/spec/plans.html#execute) step specification.

Reply:
Correct, that's exactly the page I followed when the `ResultOutcome` was added. We did not have JSON schema to accompany it, adding it here.

Reply:
Good.